www/js/services/incidents_services.js.BACKUP.5920.js

```diff
@@ -0,0 +1,316 @@
+angular.module('mobility')
```

Comment:
This file can be deleted. Looks like syntactically it won't even work with the git merge tags in it.